core/arch/arm/kernel/proc_a32.S

```diff
@@ -89,6 +89,15 @@ UNWIND(	.fnstart)
 UNWIND(	.fnend)
 END_FUNC cpu_spin_unlock
 
+/* int cpu_mmu_enabled(void) - check MMU enabled or not */
+FUNC cpu_mmu_enabled , :
```

Comment:
It's generally preferred to implement functions in c if possible. The arm32 version is easily done with help of `read_sctlr()`. For the arm64 version you'll need to add a `read_sctlr_el1()` also.